Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -3702,6 +3702,40 @@ internal unsafe bool UpdateState(InputDevice device, InputEvent* eventPtr, Input
                     stateOffsetInDevice, statePtr, stateSize, flipped);
             }
 
+            if (makeDeviceCurrent)
+            {
+                // Update the pressed/not pressed state of all buttons that have changed this update
+                // Keyboard has enough ButtonControls that it's faster to find out which have actually changed rather than test all.
+                if (InputSettings.readValueCachingFeatureEnabled || device is Keyboard)
+                {
+                    foreach (var button in device.m_UpdatedButtons)
+                    {
+#if UNITY_EDITOR
+                        if (updateType == InputUpdateType.Editor)
+                        {
+                            ((ButtonControl)device.allControls[button]).UpdateWasPressedEditor();
+                        }
+                        else
+#endif
+                        ((ButtonControl)device.allControls[button]).UpdateWasPressed();
+                    }
+                }
+                else
+                {
+                    foreach (var button in device.m_ChildrenThatAreButtonControls)
+                    {
+                        #if UNITY_EDITOR
```

Comment:
minor but different indenting to the #ifdef a few lines above

Reply:
Fair point, I'll fix - unfortunately there seems to be not much standardisation for #if alignment even within this one file, let alone the rest of the codebase 😅